src/domain/entities/Action.ts

```diff
@@ -118,7 +118,7 @@ export const getPageActions = (
 
                 const hasUserAccess = actionUsers.includes(user.id);
                 const hasUserGroupAccess = _.intersection(actionUserGroups, userGroupIds).length > 0;
-                const hasPublicAccess = action.publicAccess !== "--------";
+                const hasPublicAccess = action.publicAccess && action.publicAccess !== "--------";
```

Comment:
[No need to change] It could be `Boolean(action.publicAccess)` to make it more readable, which is because `action.publicAccess` can be undefined, otherwise it seems a little strange that both conditions are together. I have done it this way many times so never mind.